docs/features/Obsidian compatibility.md

```diff
@@ -1,6 +1,4 @@
 ---
-tags:
-  - plugin/transformer
```

Comment:
lets add the `features` tag for this, maybe /s/plugin/features for all of these tags under the folder?

Reply:
To make sure I understand you correctly: For all feature pages that previously had a plugin/*** tag, they instead should get a feature/*** tag and the new (more technical) plugin pages get the plugin/*** tags (like I already assigned)?

Reply:
yes.

Reply:
Done.